recipes-devtools/syslinux/syslinux_6.%.bbappend

```diff
@@ -1,5 +1,6 @@
-PACKAGES =+ "${PN}-isohybrid"
-PACKAGES =+ "${PN}-mboot"
+PACKAGES =+ " \
+	     ${PN}-isohybrid \"
+             ${PN}-mboot \"
```

Comment:
I'm not sure what you mean? As requested I removed the two PACKAGES=+ line and made one PACKAGE=+ over multiple lines with escapes.